FoodFriends/app/src/main/res/layout/item_restaurant.xml

```diff
@@ -4,7 +4,8 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginTop="60dp"
+    android:layout_marginTop="0dp"
+    android:layout_marginBottom="15dp"
```

Comment:
Just a comment about having these in a dimens value file, so there's a consistent padding/margin throughout the app. Material theme generally have these as theirs. https://gist.github.com/aeroechelon/f6c205e04b241b43c80b

Reply:
Have made a dimensions files, and now the text sizes and margin/padding sizes reference the file!